applications/serializers.py

```diff
@@ -147,6 +157,10 @@ def update(self, instance, validated_data):
 
         if "review_status" in validated_data:
             bootcamp_application = instance.bootcamp_application
+            if bootcamp_application.state != AppStates.AWAITING_SUBMISSION_REVIEW.value:
+                bootcamp_application.revert_decision()
```

Comment:
The intention here is to just temporarily change the application status back to `AppStates.AWAITING_SUBMISSION_REVIEW` if it isn't already, so that the methods `approve_submission` or `reject_submission` will then work normally based on the state transition criteria.  Only want to save it in this state if the submission is being waitlisted.